js/script.js

```diff
@@ -0,0 +1,110 @@
+// Mobile Menu Variables
+const containerMenu = document.querySelector('.menu');
+const closeMenu = document.querySelector('.menu-close');
+const openMenu = document.querySelector('.menu-open');
+const linkMenu = document.querySelectorAll('.menu a');
+
+// Mobile Menu Functions
+function menuOpen() {
+  containerMenu.style.display = 'flex';
+}
+function menuClose() {
+  containerMenu.style.display = 'none';
+}
+function toggleMenu() {
+  if (window.matchMedia('(min-width: 768px)').matches) {
+    containerMenu.style.display = 'flex';
+  } else {
+    containerMenu.style.display = 'none';
+  }
+}
+
+// Mobile Menu Events
+linkMenu.forEach((link) => {
+  link.addEventListener('click', menuClose);
+});
+
+openMenu.addEventListener('click', menuOpen);
+closeMenu.addEventListener('click', menuClose);
+window.addEventListener('resize', toggleMenu);
+
+// Featured Artists Variables
+
+const featContent = [
+  {
+    pic: 'assets/img/artist1.png',
+    name: 'Michael Dev',
+    music: 'Indy Rock / Punk Rock',
+    intro:
+      'Get ready for the music of sit amet consectetur adipisicing elit. Repudiandae.',
+  },
+  {
+    pic: 'assets/img/artist2.png',
+    name: 'John Java',
+    music: 'Country rock',
+    intro:
+      'Big band format sit amet consectetur adipisicing elit. Repudiandae harum nisi.',
+  },
+  {
+    pic: 'assets/img/artist3.png',
+    name: 'James Git',
+    music: 'Punk Rock',
+    intro:
+      'For the first time in America amet consectetur elit. Repudiandae adipisicing.',
+  },
+  {
```

Comment:
- [ ] Good job dynamically creating and rendering the featured speakers section(I especially like the names of the speakers 😉). It works perfectly well, you only have to make a few tweaks and you're good to go.
- The image shadow in the top left corner should be grey and white as shown in the images below.
- Reduce the font size of the italicised titles of the speakers.
- The subtitle should have a grey horizontal line beneath it, kindly add this.

_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/124092300-8ffab600-da0b-11eb-91a7-1aca95bbcd0a.png) | ![image](https://user-images.githubusercontent.com/49621279/124092465-bddffa80-da0b-11eb-903c-46e35c5611de.png)